packages/cursorless-engine/src/languages/TreeSitterQuery/TreeSitterQuery.ts

```diff
@@ -99,6 +100,9 @@ export class TreeSitterQuery {
           range: captures
             .map(({ range }) => range)
             .reduce((accumulator, range) => range.union(accumulator)),
+          allowMultiple: captures
```

Comment:
Why not `captures.some(capture => capture.allowMultiple)`?

Reply:
Oh yeah that's the way to go. Not sure how I landed here 🤔. I blame copilot 😅

Reply:
Sure blame the servants :p